.gitignore

```diff
@@ -2,3 +2,4 @@
 *.swp
 node_modules
 /dist
+.idea/
```

Comment:
```suggestion
/.idea
```
Can this work? Is there a meaning in trailing slash?

Reply:
changed!